release-notes/opensearch-release-notes-3.2.0.md

```diff
@@ -0,0 +1,898 @@
+
+
+### OpenSearch Reports
+
+
+* [AUTO] Increment version to 3.2.0-SNAPSHOT ([#1105](https://github.com/opensearch-project/reporting/pull/1105))
+
+
+### OpenSearch System Templates
+
+
+* [AUTO] Increment version to 3.2.0-SNAPSHOT ([#86](https://github.com/opensearch-project/opensearch-system-templates/pull/86))
+
+
+### OpenSearch Performance Analyzer
+
+
+* Bump spotbug to 6.2.2 and checkstyle 10.26.1 ([#826](https://github.com/opensearch-project/performance-analyzer/pull/826))
+* Increment version to 3.2.0-SNAPSHOT ([#823](https://github.com/opensearch-project/performance-analyzer/pull/823))
+
+
+### OpenSearch Query Insights
+
+
+* [AUTO] Increment version to 3.2.0-SNAPSHOT ([#380](https://github.com/opensearch-project/query-insights/pull/380))
+* [AUTO] Add release notes for 3.2.0 ([#395](https://github.com/opensearch-project/query-insights/pull/395))
+
+
+### OpenSearch Search Relevance
+
+
+* Adding template for feature technical design ([#201](https://github.com/opensearch-project/search-relevance/issues/201))
+
+
+### OpenSearch Security Analytics
+
+
+* [AUTO] Increment version to 3.2.0-SNAPSHOT ([#1552](https://github.com/opensearch-project/security-analytics/pull/1552))
+
+
+### OpenSearch Skills
+
+
+* Update the maven snapshot publish endpoint and credential ([#601](https://github.com/opensearch-project/skills/pull/601))
+* Bump gradle, java, lombok and fix ad configrequest change ([#615](https://github.com/opensearch-project/skills/pull/615))
+* Bump version to 3.2.0.0 ([#605](https://github.com/opensearch-project/skills/pull/605))
+
+
+### SQL
+
+
+* Add enforce-labels action ([#3816](https://github.com/opensearch-project/sql/pull/3816))
+* Bump gradle to 8.14 and java to 24 ([#3875](https://github.com/opensearch-project/sql/pull/3875))
+* Update commons-lang exclude rule to exclude it everywhere ([#3932](https://github.com/opensearch-project/sql/pull/3932))
+* Add release notes for 2.19.3 ([#3910](https://github.com/opensearch-project/sql/pull/3910))
+
+
+## REFACTORING
+
+
+### OpenSearch Search Relevance Dashboards
+
+
+* Code refactors + Unit tests for query\_set\_create ([#580](https://github.com/opensearch-project/dashboards-search-relevance/pull/580))
+* Code refactors + Unit tests for search\_configuration\_create ([#587](https://github.com/opensearch-project/dashboards-search-relevance/pull/587))
+* Code refactors + Unit tests for judgment\_create ([#588](https://github.com/opensearch-project/dashboards-search-relevance/pull/588))
+* Code Refactor + Unit Tests for query\_set\_listing and query\_set\_view ([#595](https://github.com/opensearch-project/dashboards-search-relevance/pull/595))
+* Code Refactor + Unit Tests for search\_configuration and judgment ([#602](https://github.com/opensearch-project/dashboards-search-relevance/pull/602))
+* Code Refactor + Unit Tests for experiment ([#613](https://github.com/opensearch-project/dashboards-search-relevance/pull/613))
```

Comment:
Not too many comments but do we want to make the `Code refactors` `Code Refactor` consistent?

Reply:
Done!